pkgs/development/libraries/dlib/default.nix

```diff
@@ -3,6 +3,7 @@
 
   # see http://dlib.net/compile.html
 , avxSupport ? true
+, cudaSupport ? false
```

Comment:
Hmm, I'm wondering if we want to add a `dlib-nvidia` package to `pkgs/top-level/all-packages.nix` which is basically `dlib.override { cudaSupport = true; }`, otherwise, users have to build `dlib` with CUDA locally.

Reply:
Thank you, you cleared things up for me. Sorry for not understanding the process before using your time.
In case it was not clear, I am new to both OSS contributing as well as nix/nixpkgs/nixos, so I appreciate your feedback and direction.

Can we reject this PR and refile under the appropriate branch, or is there some other mechanism to move it to the appropriate place?

One small touch of confusion: it's not 100% clear to me under what scenarios I should PR against release-20.03 vs. master (given your wording, is it safe to assume I can just file against master until I know of a reason to use release-20.03?)

Reply:
> Thank you, you cleared things up for me. Sorry for not understanding the process before using your time.
In case it was not clear, I am new to both OSS contributing as well as nix/nixpkgs/nixos, so I appreciate your feedback and direction.

No worries, everyone of us was a beginner at some point :) 

> Can we reject this PR and refile under the appropriate branch, or is there some other mechanism to move it to the appropriate place?

I'm afraid yes: we could change the branch to merge onto, but not your base branch (i.e. the branch from your fork with your CUDA changes).

> One small touch of confusion: it's not 100% clear to me under what scenarios I should PR against release-20.03 vs. master (given your wording, is it safe to assume I can just file against master until I know of a reason to use release-20.03?)

Basic criteria is listed here: https://gist.github.com/grahamc/c60578c6e6928043d29a427361634df6#what-to-backport (you may want to take a look at this as well: https://nixos.wiki/wiki/Maintainers:Maintainers)

But those are no hard rules, so in case you have a good reason, feel free to open a PR anyway to discuss :)

Reply:
Thank you very much again. I appreciate your time greatly.